core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.spec.ts

```diff
@@ -38,13 +39,15 @@ describe('Contributor dashboard page', function() {
     can_review_voiceover_for_language_codes: ['en', 'pt', 'hi'],
     can_review_questions: true
   };
+  var windowRef = new WindowRef();
 
   importAllAngularServices();
 
   beforeEach(angular.mock.module('oppia', function($provide) {
     var ugs = new UpgradedServices();
     for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
       $provide.value(key, value);
+      $provide.value('WindowRef', windowRef);
```

Comment:
Why is this in the loop? Can we move remove ugs and use importAllAngularServices?

Reply:
@chris7716 is this comment resolved?

Reply:
@DubeySandeep I am working on it.

Reply:
Done